src/includes/capture-error/javascript.mdx

```diff
@@ -9,3 +9,5 @@ try {
   Sentry.captureException(err);
 }
 ```
+
+Learn more about how to manually capture errors or enable message capture with Sentry's JavaScript SDK in [our documentation on Usage](/platforms/javascript/usage/).
```

Comment:
One concern we might have here is javascript can be included in other pages, so this would link to the wrong platform.

We could embed a `<PlatformLink>` to solve for that:

https://docs.sentry.io/contributing/pages/components/#platformlink

```
<PlatformLink to="/usage/">our documentation on Usage</PlatformLink>
```

(didnt test but should work)

Reply:
Thanks. I'll give that a go.